examples/upload_disk.py

```diff
@@ -103,6 +103,13 @@ def parse_args():
              "smaller number of workers you may want use larger value."
              .format(client.BUFFER_SIZE))
 
+    parser.add_argument(
+        "--inactivity-timeout",
+        type=int,
+        help="Number of seconds to wait before aborting inactive "
```

Comment:
an inactive

Reply:
@bennyz fixed in https://github.com/oVirt/python-ovirt-engine-sdk4/compare/54dddec8406737f371a39a2658fbc4e01896001e..bed250dec78b23b7c151fc0091c758851f8d8959